src/main/java/io/deephaven/csv/densestorage/QueueWriter.java

```diff
@@ -42,8 +42,8 @@
     private TARRAY genericBlock;
     /**
      * Start of the current block. This is typically 0, but not always. If the caller does an early flush (before the
-     * block is filled), you can have multiple linked list nodes sharing different segments of the same underlying block
-     * storage.
+     * block is filled), you can have multiple linked list nodes sharing different segments (slices) of the same
+     * underlying block storage.
```

Comment:
Is the word "storage" there still relevant?  Asking because the phrase "block storage" suggests a different reading, and a different concept potentially, in this cloud day and age.